packages/rocketchat-language-plugin/server/add_or_update_user.js

```diff
@@ -0,0 +1,26 @@
+Meteor.methods({
+	add_or_update_user(channel, plugin_name) {
```

Comment:
This does not have to be a Meteor method because it is not called from the client side.